components/Events/Filters.js

```diff
@@ -0,0 +1,142 @@
+            <div className={styles.inner}>
+                <div>
+                    <input
+                        placeholder='Search Events'
+                        className={styles.searchbar}
+                        value={searchValue}
+                        onChange={(e) => setSearchValue(e.target.value)}
+                        onKeyPress={(e) => e.key === 'Enter' && handleSearch()}
+                        />
+                    <button onClick={handleSearch} className={styles.searchbutton}>search</button>
+                    <button onClick={() => setFilterOpen(!filterOpen)} className={styles.searchbutton}>
+                        filter
+                        <FontAwesomeIcon className={styles.arrow} icon={filterOpen ? faChevronUp : faChevronDown}/>
+                    </button>
+                </div>
+                {filterOpen &&
+                <div className={styles.filteroptions}>
+                    <div>
+                        <h3 className={styles.header}>Committees </h3>
+                        {Object.keys(chosenComms).map((comm) => (
+                            <div key={comm}>
+                                <input type="checkbox" id={comm} checked={chosenComms[comm]} onChange={() => handleCommChange(comm)}/> {comm}
+                            </div>
+                        ))}
+                    </div>
+                    <div>
+                        <h3 className={styles.header}>Days</h3>
+                        {days.map((day,index) => (
+                            <div key={day}>
+                                <input type="checkbox" id={index} checked={chosenDays[index]} onChange={() => handleWeekdayChange(index)}/> {day}
+                            </div>
+                        ))}
+                    </div>
+                    <div>
+                        <h3 className={styles.header}>Time</h3>
+                        <div className={styles.timecontainer}>
+                            <div>
+                                <div>From </div>
+                                <div>To </div>
+                            </div>
+                            <div>
+                                <input type='time' className={styles.timeinput} value={fromTime} onChange={(e) => setFromTime(e.target.value)} />
+                                <br/>
+                                <input type='time' className={styles.timeinput} value={toTime} onChange={(e) => setToTime(e.target.value)} />
+                            </div>
+                        </div>
+                    </div>
+                    <div>
+                        <h3 className={styles.header}>Location</h3>
+                        <input type="checkbox" id="online" checked={chosenLoc.online} onChange={() => handleLocationChange('online')}/> Online <br/>
+                        <input type="checkbox" id="in_person" checked={chosenLoc.in_person} onChange={() => handleLocationChange('in_person')}/> In-Person
```

Comment:
Can use `Object.map` so that if, for instance, someone modifies the L135, they won't forget to modify the other line (since they will be collapsed together)